integration_tests/pages/sexPage.ts

```diff
@@ -0,0 +1,25 @@
+import ImprisonmentStatusPage from './imprisonmentStatus'
+import Page, { PageElement } from './page'
+
+export default class SexPage extends Page {
+  constructor() {
+    super('What is their sex?')
+  }
+
+  static goTo(id: string, expectRedirect = false): SexPage | ImprisonmentStatusPage {
+    cy.visit(`/prisoners/${id}/sex`)
+    return expectRedirect ? Page.verifyOnPage(ImprisonmentStatusPage) : Page.verifyOnPage(SexPage)
```

Comment:
probably just worth having two methods for this so you don't have to worry about casting..